cosmic-core/server/src/main/java/com/cloud/network/NetworkServiceImpl.java

```diff
@@ -996,6 +1001,45 @@ public Network createGuestNetwork(final CreateNetworkCmd cmd) throws Insufficien
         return network;
     }
 
+    public void checkIpExclusionList(final String ipExclusionList, final String cidr, List<NicVO> nicsPresent) {
+        if (!org.apache.commons.lang.StringUtils.isEmpty(ipExclusionList)) {
```

Comment:
Any reason for using the full qualified name of the class? Is there another `StringUtils` already being used?